Modules/FT0/multinode-config.json

```diff
@@ -0,0 +1,106 @@
+{
```

Comment:
Hi, I saw on todays slides by Alla that you had a multinode workflow where you merge *sampled data*, instead of *monitor objects*. Is that really true, or that was some kind of mistake in slides?

Reply:
To be honest I'm a bit confused, because from generated graph https://i.imgur.com/sCPfHVe.jpg (using -g option) there is no task execution on local node (which would create monitoring object), but merger works on monitoring objects, so input of merger should be a monitoring object. Well I need to look on implementation because it looks like I'm missing something.

Reply:
Let us know if you need some clarifications.

Reply:
See the slide 5, this is how it is supposed to look like in the big picture:
 https://indico.cern.ch/event/873380/contributions/3850119/attachments/2036439/3409783/WP7plenaryMay.pdf
Some parallel proxies are replaced with one, as far as I remember, but otherwise it should match what you get (disregarding devices generated automatically by DPL).

Reply:
@knopers8 
Thank you for the slide with explanation.
Right it seems I made a mistake in a slide, there is a task execution and monitoring object creation before sending it to remote device, but I do not undestand why the generated graph does not contain task runner.
Maybe adding -g option ends workflow generation before attaching task runner to workflow?

Reply:
I doubt it. I've just crosschecked on my machine and the proxy after dispatcher is still created. 

Does your machine name (or the --host argument) match one from your list of machines in a data sampling policy? A proxy for sampling policies won't be created on machines which don't need it.

Reply:
@knopers8 Well I will update multinode-config to my current version, because I completely removed data sampling policies + remote task.

Reply:
Yes, so to make your local task BasicDigitQcTask appear, you have to run QC with `--local --host 2a` (or 3a), so QC knows that this is the proper machine to load this task. I am not sure however, how far can you get away with mocking the names instead of using real machines.